pkg/image/registry/imagestreamimport/rest.go

```diff
@@ -104,33 +106,37 @@ func (r *REST) Create(ctx apirequest.Context, obj runtime.Object, _ bool) (runti
 	if !ok {
 		return nil, kapierrors.NewBadRequest("unable to get user from context")
 	}
-	isCreateImage, err := r.sarClient.Create(authorizationapi.AddUserToSAR(user,
-		&authorizationapi.SubjectAccessReview{
-			Action: authorizationapi.Action{
-				Verb:     "create",
-				Group:    imageapi.GroupName,
-				Resource: "images",
+	createImageSAR := authorizationutil.AddUserToSAR(user, &authorizationapi.SubjectAccessReview{
+		Spec: authorizationapi.SubjectAccessReviewSpec{
+			ResourceAttributes: &authorizationapi.ResourceAttributes{
+				Namespace: inputMeta.Namespace,
```

Comment:
Was this namespace scoped before?  `images` are cluster scoped.

Reply:
This is imagestreamimport, not image. Iow. you create a resource that tells the server to import some data inside particular image stream.

Reply:
I take that back. Yes, you're right this should not be namespaced.